src/main/kotlin/it/pagopa/wallet/documents/wallets/Application.kt

```diff
@@ -11,13 +12,15 @@ data class Application(
     val id: String,
     val name: String,
     val status: String,
-    val lastUpdateDate: String
+    val lastUpdateDate: String,
+    val metadata: Map<String, String>
```

Comment:
Isn't it better to have **details** ?

Reply:
good point, the idea is to have a general data related to a wallet. `details` implies strong typing, so fixed property and not a generic property with `Map`

Reply:
Yes yes in fact I was only talking about the name of the attribute of the class.
```suggestion
    val details: Map<String, String>
    
```